.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,47 @@
+name: integration-tests
+
+env:
+  AWS_REGION: us-east-1
+  AWS_ROLE_DURATION: 3600
+  # role generated from https://github.com/Sceptre/sceptre-aws/blob/master/config/prod/gh-oidc-sceptre-tests.yaml
+  AWS_ROLE: arn:aws:iam::743644221192:role/gh-oidc-sceptre-tests
+
+on:
+  workflow_run:
```

Comment:
I think there are problems with setting up [workflow dependencies](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_run).

Even when setting the [github.event.workflow_run.conclusion](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow) flag it does not gather up conclusions from all dependent workflows as suggested in this discussion  https://github.com/orgs/community/discussions/26238